src/data/resume-data.types.ts

```diff
@@ -0,0 +1,64 @@
+import { StaticImageData } from "next/image";
+export type ResumeData = {
```

Comment:
having that adds pretty much no value but adds a lot of overhead In maintaining the type. using `as const` ensures we have a type to check against and I'd rather not include this change because of that.

Reply:
The reason I added types is because exporting as const won't allow as much customization.

For Example: 
If I have any desc with a field `customBullet` and i use it in the `page.tsx` it works fine, because the field exists. 

if I remove the field from the config, we get this error `Property 'customBullet' does not exist on type 'never'.` 

The app still works fine, but Vercel doesn't allow this. 

and for possibility of making the desc both array or string, exporting as const won't allow it unless there's atleast one array and one string in the whole config, or it'll have the type never, and the deployment fails.